media-libs/shaderc/shaderc-2017.2.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+COMMIT_ID="7a23a01742b88329fb2260eda007172135ba25d4"
```

Comment:
Please name it `EGIT_COMMIT` to make it easier to switch to git-r3 if necessary.

Reply:
Why one would want to switch to git-r3 and still be stuck to a particular commit?
This is versioned ebuild, it won't have git-r3.

Reply:
Do you really have that much of a problem with using consistent variable names?

Reply:
No, but still rationale 'in case someone switches versioned ebuild to git-r3' sounds weird to me.

Reply:
Gentoo people do all weird things. Switching to git-r3 locally to check what happens if '.git' is there.